### ISSUE TITLE
Please provide the documentation

### ISSUE THREAD

**Issue by netzgestaltung** [A5hleyRich/delightful-downloads#170]
Hi there, the page "delightfuldownloads.com" is down for a time now.

There are still 10000+ active installations, so it would be helpful to get the documentation of it in some place. 

i am willing to provide it but currently i just try to make my own customizations to the plugin

kind regards
tom

**Comment by obewanz** [comment]
Agreed, can't find the documentation, so now I'm looking to replace this plugin on all four of my websites.

**Comment by jodzeee** [comment]
@svenbolte forked it and has kept it updated.
